selendroid-server/AndroidManifest.xml

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="0.17.0-SNAPSHOT" package="io.selendroid.server">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:versionCode="1"
+    android:versionName="0.17.0-SNAPSHOT"
+    package="io.selendroid.server">
 
-    <uses-sdk android:minSdkVersion="10"/>
+    <uses-sdk android:minSdkVersion="18"/>
```

Comment:
Selendroid is intended to work on lower API levels so we can't just set the min API to 18.